README.md

```diff
@@ -46,6 +46,27 @@ access." You can now run `siac` in a separate command prompt to interact with
 siad. From here, you can send money, mine blocks, upload and download
 files, and advertise yourself as a host.
 
+Building From Source
+--------------------
+
+To build from source, [Go 1.6 must be installed](https://golang.org/doc/install)
+on the system. Then simply use `go get`:
+
+```
+go get -u github.com/NebulousLabs/Sia/...
+```
+
+This will download the Sia repo to your `$GOPATH/src` folder, and install the
+`siad` and `siac` binaries in your `$GOPATH/bin` folder.
+
+To stay up-to-date, run the previous `go get` command again. Alternatively, you
+can use the Makefile provided in this repo. Run `git pull origin master` to
+pull the latest changes, and `make release-std` to build the new binaries. You
```

Comment:
so now the question... what do we do to prevent other users from running into this problem? Do we remove the `-a` flag, or is there something more subtle? I do want to make sure that anyone running `make release` is going to be getting a fully updated binary.